keycloak/realm_user_profile.go

```diff
@@ -65,5 +65,50 @@ func (keycloakClient *KeycloakClient) GetRealmUserProfile(ctx context.Context, r
 		return nil, err
 	}
 
+	for _, attr := range realmUserProfile.Attributes {
+		if attr.Validations != nil {
+			for name, config := range attr.Validations {
+
+				c := make(map[string]interface{})
+				for k, v := range config {
+					if _, ok := v.([]interface{}); ok {
```

Comment:
I know that the validators that come bundled with Keycloak either have string or array keys (which is what we're checking for here), but do you think it would also be useful to handle nested objects?  You mention that you have custom validators, do any of them use nested objects?